pycds/orm/manual_matviews/__init__.py

```diff
@@ -19,21 +19,9 @@
 export the latest version. When a PyCDS release is created, it will "freeze"
 this set of views. Following any PyCDS release, further migrations and further
 releases will "freeze" later sets of views.
+
+At present all manual matviews have been updated to native matviews, and this
+file is residual.
```

Comment:
Is there a reason we can't delete the file altogether?

Reply:
I thought it might be needed to support upgrading and downgrading?